### ISSUE TITLE
nav_p does not respond to joystick command, reason unknown

### ISSUE THREAD

**Issue by sunava** [refills-project/refills_first_review#2]
@airballking @amaldo @ichumuh what happned:

When scanning the bottom markers, the robot just stopped reacting to joystick commands. No clear error msg on the console identified the problem. Start stopping the run-stop and wiggling some cables that were close to the laser fixed the problem.

Suggested solution have meaningful error or diagonstic msg from navigation.

**Comment by ichumuh** [comment]
> Start stopping the run stop and wiggling some cables that were close to the laser fixed the problem.

Pressing and releasing the emergency stop alone didn't fix the problem? Sometimes the emergency stop randomly kicks in. 

> Suggested solution have meaningful error or diagonstic msg from navigation.

Yes we need this.

**Comment by airballking** [comment]
> > Start stopping the run stop and wiggling some cables that were close to the laser fixed the problem.

> Pressing and releasing the emergency stop alone didn't fix the problem? Sometimes the emergency stop randomly kicks in.

No, pressing and releasing the e-stop did not fix the problem. We first tried that, then restarted all the launch-files. Still, no go. Then, Alexis suggested to look at the cables. ;)
